app/players/elves/elf.py

```diff
@@ -0,0 +1,18 @@
+from abc import ABC
+from app.players.player import Player
+
+
+class Elf(Player, ABC):
```

Comment:
Do not repeat abstract methods from a parent if you have an abstract class that inherits another abstract class.
So `Player` already child of `ABC` class , so you can interhance just from `Player`

Reply:
Sure, got it

Reply:
I did it only because my IDE show me an warning: "Class Elf must implement all abstract methods", so i think that would be correct to duplicate ``ABC`` class to pass requirement to impelent abstract functions to further classes